docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -121,6 +121,7 @@ spec:
 | timeout | duration | The maximum length of time to execute deployment before giving up. Default is 6h. | No |
 | notification | [DeploymentNotification](#deploymentnotification) | Additional configuration used while sending notification to external services. | No |
 | postSync | [PostSync](#postsync) | Additional configuration used as extra actions once the deployment is triggered. | No |
+| vpcConfig | [VPCConfig](#vpcconfig) | Additional configuration used to link to specific vpc. | No |
```

Comment:
@sivchari I mean instead of adding it here, only make the vpc configuration available via Function manifest file. I think we should keep this configuration file contains generic configurations only.

Reply:
@khanhtc1202 
Ah, sorry. I fixed it. Like this ?